### ISSUE TITLE
Feature request: Add option to purge blob from Azure CDN

### ISSUE THREAD

**Issue by crackalak** [Microsoft/AzureStorageExplorer#334]
**Storage Explorer Version**: 1.2.0
**Platform/OS Version**: Windows 10
**Architecture**: ia32
**Regression From**: 

#### Expected Experience: ####
When selecting a blob in a storage account that has Azure CDN attached, be able to purge the blob from the CDN.

**Comment by MRayermannMSFT** [comment]
During 1.5.0 let's look into this and see if it makes sense in Storage Explorer.

**Comment by MRayermannMSFT** [comment]
Hey @crackalak , I did some digging into Azure CDN, and I'm not 100% on what scenario you are talking about? Can you point me to something that talks about purging blobs from CDN? Thanks!

**Comment by crackalak** [comment]
Hi @MRayermannMSFT , please see https://docs.microsoft.com/en-us/azure/cdn/cdn-purge-endpoint and the API info from here https://docs.microsoft.com/en-us/rest/api/cdn/endpoints/purgecontent

**Comment by MRayermannMSFT** [comment]
Hey @crackalak , if you had the ability to add this yourself to Storage Explorer (either via the project being open source, or, writing an "extension") would you be interested in doing it?

**Comment by crackalak** [comment]
Hi @MRayermannMSFT , yes I would be interested in doing it :)

**Comment by crackalak** [comment]
Yes that would be a good start, a setting to purge automatically on overwrite of a blob would be also be nice
